dashboard/lib/services/curriculum_pdfs.rb

```diff
@@ -121,6 +121,39 @@ def self.get_pdfless_lessons(script)
       end
     end
 
+    def self.regenerate_pdfs(scripts)
```

Comment:
just checking my understanding: this is just for manual use from a console?

Reply:
Yep!